lib/rspec/core/notifications.rb

```diff
@@ -414,6 +414,24 @@ def duplicate_rerun_locations
           end
         end
       end
+
+      def rspec_shell_command
+        @rspec_shell_command ||= begin
+          if spring_rspec?
+            'bin/rspec'
+          else
+            'rspec'
+          end
+        end
+      end
+
+      def spring_rspec?
+        lockfile     = Bundler::LockfileParser.new(Bundler.default_lockfile.read)
```

Comment:
We can't use Bundler here, we don't have a dependency on it and our CI enforces that on purpose.